internal/expr/query.go

```diff
@@ -69,9 +71,9 @@ func (pe *PreparedExpr) Query(args ...any) (ce *QueryExpr, err error) {
 		}
 	}
 
-	// Query parameteres.
-	qargs := []any{}
-	for i, typeMember := range pe.inputs {
+	// Query parameters.
```

Comment:
Not for this patch, but this is an example of what I mentioned in our meeting. If you are using comments to label the steps of a large method, just make the steps their own methods.